### PR TITLE
Refactor assert revert helper to encapsulate promises

### PR DESCRIPTION
**Pull request by facuspagnuolo**

### CODE REVIEW

File: test/helpers/assertRevert.js [hunk 1]
```diff
@@ -1,3 +1,10 @@
-module.exports = function (error) {
-  assert.isAbove(error.message.search('revert'), -1, 'Error containing "revert" must be returned');
+export default async promise => {
+  try {
+    await promise;
+    assert.fail('Expected revert not received');
+  } catch (error) {
+    const revertFound = error.message.search('revert') >= 0;
+    console.log('Revert found: ', revertFound);
```
Comment (federicobond) [inline test/helpers/assertRevert.js]: Remove this `console.log`.

Comment (facuspagnuolo) [reply test/helpers/assertRevert.js]: :+1:

**Comment by facuspagnuolo** [comment]
@federicobond I change rebased this PR from master so it can be merged before #627. Actually #627 is not depending on this PR

**Comment by federicobond** [comment]
Please remove the `.node-xmlhttprequest-sync-19998` file and move the `.idea` gitignore rule to your own `.git/info/exclude`.

Everything else looks good! 🎄

**Comment by facuspagnuolo** [comment]
done @federicobond :)

**Comment by janezkranjc** [comment]
will not the catch also catch the assert.fail('...')?

**Pull request merged**